src/example-types/esTwoLevelAggregation.js

```diff
@@ -30,19 +36,17 @@ module.exports = {
               context.key_data
             )
           ),
-          aggs: {
-            twoLevelAgg: {
-              [context.value_type]: _.omitBy(
-                _.isNil,
-                _.extend(
-                  {
-                    field: context.value_field,
-                  },
+          aggs: F.arrayToObject(
+            _.identity,
+            metric =>
+              _.omitBy(_.isNil, {
+                [metric]: _.extend(
+                  { field: context.value_field },
                   context.value_data
-                )
-              ),
-            },
-          },
+                ),
+              }),
+            _.size(context.include) ? context.include : [context.value_type]
```

Comment:
This means that `include: []` will include everything instead of nothing. We definitely need to document that

Reply:
Ok